.github/workflows/python.yaml

```diff
@@ -142,7 +142,7 @@ jobs:
           MODULE: ${{ matrix.module }}
         run: |
           if [ "$MODULE" == "gpu" ]; then
-            ${{ env.SELF_HOST_PYTHON }} -m pip install -U "openai>1,<1.40"
```

Comment:
How about set it to `openai>1.40`? or it won't update the openai library we may not find issue.